### ISSUE TITLE
Set up separate staging repo to test changes

### ISSUE THREAD

**Issue by fsteeg** [hbz/geodata#34]
Should be done on quaoar1, thus this depends on #23

**Comment by fsteeg** [comment]
A separate staging repo was set up by @SBRitter already. Next step: set up monitoring with monit.

**Comment by fsteeg** [comment]
Ah, no, we have a separate issue for that: #29, closing this.
